students/20th/ahnjunghyun/style/login.css

```
@@ -0,0 +1,59 @@
+* {
+  box-sizing: border-box;
+}
+
+div.outlineBox {
```

이렇게 선택하시면 div중에서 outliinebox라는 클래스명을 가진 html 요소를 선택한다는 의미인데,
```suggestion
.outlineBox {
```
이런식으로 그냥 클래스명으로만 선택하셔도 무방해보입니다!

다른부분도 전체적으로 마찬가지로 보이네요,
해당 클래스명을 가진 요소들중에 특정 태그만 선택하고 싶은게 아니라면 모두 불필요해보입니다!
이부분 이해안되시면 찾아와서 질문해주셔도 됩니다~!😀